test/NetCore.Tests/NetCore.Tests.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
+   <RuntimeFrameworkVersion>2.0.3</RuntimeFrameworkVersion>
```

Comment:
hmmm, @attilah do you have a recommendation to avoid this? Because we are installing 2.0.3 if nothing is installed already, but if it is, then not sure whether this would break...

Reply:
I'd love to use a different solution if anyone has an idea of where to look.